contributing/team-communication.txt

```diff
@@ -11,44 +11,32 @@ find and track it may take some time.
     access to these automatically but you can also request a log-in as an
     external collaborator.
 
-Jabber: Instant messaging
--------------------------
-
-On a day-to-day level, the team meets in a Jabber chatroom (see the list of
-:team_plone:`addresses <addresses>` for the specifics). You are
-welcome to join via your own Jabber account, but one will likely be
-provided to you, in which case that account should be preferred. To
-get started using Adium:
-
-
-* :menuselection:`File --> Join Group Chat` (Select your GTalk or Jabber Account in the drop-down menu)
-* Use either your GTalk or LifeSci LDAP Account Credentials
-* Enter the "Chat Room Name" and "Server"
-
-
-.. _jabber_screenshot:
-.. figure:: /images/jabber_screenshot.png
-   :align:  center
-
-The daily stand-up meeting is managed via this chatroom, with notes in google
-docs that are edited throughout the day as people complete the tasks assigned
-to them.
-
-Slightly less frequently, the team meets on Skype, Google hangouts, or most
-frequently, Zoom for voice discussions. These subgroup, or “mini group”,
-meetings are organized as needed, but should provide feedback (tickets, notes,
-etc). More information is available under :community_plone:`conference-calls
-<minutes/conference-calls>`.
-
-Being online in Skype is not required but it is often beneficial
-to have it on during the day. Regardless, everyone should have a Skype
-account. Register and download software from http://www.skype.com.
-
-.. note:: your Skype account name along with all of your contact details
-    should be added to DevContactList on Google Docs. See below.
-
-The four major development tools that the OME project runs are Trac, the
-developer documentation, Jenkins, and GitHub.
+Instant messaging and video conferencing
+----------------------------------------
+
+On a day-to-day level, the team meets in a Slack chatroom (see the list of
+:team_plone:`addresses <addresses>` for the specifics). Slack can be used in
+your internet browser or via an app; you will be invited to join the team by
+an admin.
+
+The daily stand-up meeting is managed via the '#general' channel, with notes
+in google docs that are edited throughout the day as people complete the tasks
+assigned to them.
+
+Slightly less frequently, the team meets on Zoom for voice discussions. These
+subgroup, or “mini group”, meetings are organized as needed, but should
+provide feedback (tickets, notes, etc). More information is available under
+:community_plone:`conference-calls <minutes/conference-calls>`.
+
+Other IM tools
+^^^^^^^^^^^^^^
+
+Slack is the only IM tool used by the entire OME team. Some team members do
+also use IRC (#ome on irc.freenode.net) but in general, there is no official
+support via IM and all external requests for help should be submitted and
```

Comment:
This doesn't capture for me how this is being used. I think it is a _support_ channel, but all significant conversations must be captured in something more formal (trac, trello).